test/testbase.py

```diff
@@ -78,7 +89,7 @@ async def setup():
             try:
                 os.environ['SM_ALLOWALLACCESS'] = '1'
                 set_full_access(True)
-                db = MySqlContainer('mariadb:latest')
+                db = MySqlContainer('mariadb:10.8.3')
```

Comment:
Should we be hard coding the mariadb version here? Or is there somewhere else we can specify it for metamist that this function can use?

Reply:
It's a good point, I'm really not sure. Maybe @vivbak might have an opinion, our mariadb instance is very decoupled from infrastructure code. Our current server version is `10.5.15-MariaDB-1:10.5.15+maria~buster`, so I guess we should really just test against the production code. 

Raised an issue here: https://github.com/populationgenomics/sample-metadata/issues/293